html_css/intermediate_html/html-tables.md

```diff
@@ -30,6 +30,9 @@ In this codepen example we created a table element, containing two table row ele
 </div>
 
 ### Additional Resources
+This section contains helpful links to other content. It isn’t required, so consider it supplemental for if you need to dive deeper into something.
+
+- <a href="https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables">Pencil & Paper</a> made this great article about things you can acheive with tables if you dare to spend some time giving them some love. They suggest some good habits on how you should format your data which is pretty simple and makes a huge difference. These are not a list of must does to a table, but ideas worth keeping in mind for the next time a table is bothering you with how it looks. 
```

Comment:
Rather than using an HTML link we need to use a markdown link here.

Also, I would like to see the link text used be more descriptive with regards to what the linked resource is. Check out [this resource about link text and accessibility by WebAIM](https://webaim.org/techniques/hypertext/#screen-readers-and-links) for some more information.

Reply:
would you like me to change up the other links in this lesson as well? All of them seem to be html anchor-tags...

I think this looks better regarding the descriptiveness of the link, thanks. 
![Screenshot 2022-02-06 at 01 34 18](https://user-images.githubusercontent.com/8139114/152663707-4afff461-80a3-49a8-8950-a70cd7a2d6b9.png)

Reply:
> would you like me to change up the other links in this lesson as well? All of them seem to be html anchor-tags...

Yes if you don't mind reformatting all of the other links (with the exception of knowledge check links which need a custom class to be applied) to also be markdown that would be great!

Your new link text is 🔥